### ISSUE TITLE
selector-pseudo-class-case

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#1059]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Both valid:

```css
body:HOVER {
    background-color: red;
}
```

```css
body:hover {
    background-color: red;
}
```

**Comment by jeddy3** [comment]
Sgtm.

["Like all CSS keywords, pseudo-class names are ASCII case-insensitive"](https://drafts.csswg.org/selectors-4/#pseudo-classes).

- Name: `selector-pseudo-class-case`
- Primary options: `"upper"|"lower"`
- Secondary options: none
- Message: `Expected "${actual}" to be "${expected}" (selector-pseudo-class-case)`.

We'll need to [account for](https://github.com/stylelint/stylelint/tree/master/src/rules/selector-pseudo-element-colon-notation#selector-pseudo-element-colon-notation) single colon notation for these pseudo elements:  `:first-line`, `:first-letter`, `:before` and `:after`. They should be ignored by this rule.

**Comment by evilebottnawi** [comment]
Merged. Close.

**Comment by ntwb** [comment]
Fixed via https://github.com/stylelint/stylelint/commit/8ca121c8f85c2f97b61618d7cb14107865c9b4b4
